microsoft-edge/toc.yml

```
@@ -1314,13 +1314,20 @@
           items:
             - name: Test upcoming APIs and features
               href: webview2/how-to/set-preview-channel.md
-    
+
+            - name: Prerelease testing using preview channels
+              href: webview2/how-to/prerelease-testing.md
+
+            - name: Self-hosting by deploying preview channels
+              href: webview2/how-to/self-hosting.md
```

```suggestion
            - name: Test using preview channels
              href: webview2/how-to/prerelease-testing.md

            - name: Self-host by deploying preview channels
              href: webview2/how-to/self-hosting.md
```

Consider aligning with the other nearby items.

---

i plan to change these titles in all the usual spots

---

I would want to keep the terminology `pre-release testing`. `Self-host` is fine however

---

hits in repo:
pre-release - 4 hits in 2 files (wv2) - changed them to "prerelease"
prerelease - 1389 hits in 25 files
@victorhuangwq 

keeping "Prerelease testing using preview channels", no action required.  @captainbrosset